tools/webpack.config.js

```diff
@@ -76,6 +76,10 @@ const config = {
           process.env.RXP_HTML_SRT === "true" :
           process.env.RXP_HTML_SRT !== "false",
 
+        METAPLAYLIST: isBarebone ?
+        process.env.RXP_METAPLAYLIST === "true" :
+        process.env.RXP_METAPLAYLIST !== "false",
```

Comment:
I would prefer it deactivated by default but activated only if the ENV RXP_METAPLAYLIST is set to true:
```
process.env.RXP_METAPLAYLIST === "true"
```
Still, activate it by default on the demo (as it's already done here)